vsintegration/tests/UnitTests/InlineParameterNameHintTests.fs

```diff
@@ -159,17 +159,31 @@ let wrapped = WrappedThing 42
     Assert.IsEmpty(result)
 
 [<Test>]
-let ``Hints are not (yet) shown for dicrimanted unions`` () =
+let ``Hints are shown for discriminated unions`` () =
     let code = """
 type Shape =
-    | Square of side : float
-    | Circle of radius : float
-
-let circle = Circle 42
+    | Square of side: int
+    | Rectangle of width: int * height: int
+    | Triangle of side1: int * int * side3: int
+    | Circle of int
+ 
+let a = Square 1
+let b = Rectangle (1, 2)
+let c = Triangle (1, 2, 3)
+let d = Circle 1
```

Comment:
Probably personal preference, I'd rather have these as separate negative tests with clear names, like "Hints are not shown for generated names" or "Hints are not shown for DU without field names" or something along those lines.